src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -92,6 +104,7 @@ class RsHighlightingAnnotator : AnnotatorBase() {
             } else {
                 null
             }
+            // XXX maybe now unnecessary; parent is RsMacroBinding -> RsColor.MACRO
```

Comment:
this code still necessary because there are a lot of cases when `parent is RsNameIdentifierOwner` except `RsMacroBinding`, so let's drop this comment